core/modules/menu/menu.admin.inc

```diff
@@ -724,3 +724,58 @@ function menu_reset_item_confirm_submit($form, &$form_state) {
   backdrop_set_message(t('The menu link was reset to its default settings.'));
   $form_state['redirect'] = 'admin/structure/menu/manage/' . $new_item['menu_name'];
 }
+
+/**
+ * Menu callback; global settings form for menus.
+ */
+function menu_settings_form($form, &$form_state) {
+  $form['#config'] = 'system.core';
+
+  $form['menu_breakpoint'] = array(
+    '#type' => 'radios',
+    '#title' => t('Breakpoint'),
+    '#description' => t("The 'min-width' for the menu breakpoint. Screens smaller than this may display a menu toggle button, depending on the menu-specific settings."),
+    '#options' => array(
+      'default' => t('Default (48em)'),
+      'custom' => t('Custom'),
+    ),
+    '#default_value' => config_get('system.core', 'menu_breakpoint'),
+  );
+  $form['menu_breakpoint_custom'] = array(
+    '#type' => 'textfield',
+    '#description' => t("Enter a valid CSS width value; a number followed by a unit (e.g.: '768px')."),
+    '#default_value' => config_get('system.core', 'menu_breakpoint_custom'),
+    '#size' => 8,
+    '#field_prefix' => t('min-width:'),
```

Comment:
This is a CSS property, so shouldn't be translated:

```suggestion
    '#field_prefix' => 'min-width:',
```

Reply:
Good call, fixed.